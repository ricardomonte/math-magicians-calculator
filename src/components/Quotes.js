import React from 'react';

function Quotes() {
  return (
    <div className="Quote">
      <figure>
        <blockquote>
          &ldquo;Pure mathematics is, in its way, the poetry of logic ideas&rdquo;
        </blockquote>
        <figcaption>-Albert Einstein-</figcaption>

      </figure>
    </div>
  );
}

export default Quotes;
