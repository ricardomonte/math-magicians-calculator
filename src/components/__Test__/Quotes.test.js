import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Quotes from '../Quotes';

test('return content from Home matches snapshot', () => {
  const content = render(<Quotes />);
  expect(content.container).toMatchSnapshot();
  expect(screen.getByText('-Albert Einstein-')).toBeInTheDocument();
});
