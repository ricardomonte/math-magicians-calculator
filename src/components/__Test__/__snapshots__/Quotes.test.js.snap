// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return content from Home matches snapshot 1`] = `
<div>
  <div
    class="Quote"
  >
    <figure>
      <blockquote>
        “Pure mathematics is, in its way, the poetry of logic ideas”
      </blockquote>
      <figcaption>
        -Albert Einstein-
      </figcaption>
    </figure>
  </div>
</div>
`;
